packages/portal/src/app/rentalpage/comment-generator.ts

```diff
@@ -0,0 +1,66 @@
+//import a way to get random numbers
```

Comment:
can you convert this file into an [Angular service](https://angular.io/guide/architecture-services)?